packages/cli/src/util/output/create-output.ts

```diff
@@ -35,6 +39,16 @@ export class Output {
   };
 
   print = (str: string) => {
+    if (this.textOnly) {
+      const emojis = Object.values(emojiLabels).map(emoji => `${emoji}  `);
+      const regExp = new RegExp(emojis.join('|'), 'gi');
+      const textOnlyStr = str.replace(regExp, '');
```

Comment:
I like how you are accounting for the two spaces we add in `prependEmoji`, but I'd rather match strictly on the emoji and then trim leading whitespace from the string.

Reply:
@Ethan-Arrowood Thank you for comment. I think that's much better and safety, I will apply it.

Reply:
[be185fb](https://github.com/vercel/vercel/pull/8826/commits/be185fb0a71a1e8a84a87775beee1b323bc8eb63) applied it.